npm/puppeteer/tsconfig.json

```diff
@@ -16,7 +16,7 @@
     "types": [
       "cypress",
       "./support",
-      "./node_modules/@types/node"
+      "node"
```

Comment:
To address this error: https://app.circleci.com/pipelines/github/cypress-io/cypress/59971/workflows/b10cb9c6-89ad-4a6b-8ae8-9ff9d8d6121e/jobs/2495052

![Screenshot 2024-02-22 at 11 53 39 AM](https://github.com/cypress-io/cypress/assets/1271364/d0de596d-c65c-4063-ade9-826e4a4f262c)